src/gui/aboutdialog.cpp

```diff
@@ -45,7 +45,7 @@ AboutDialog::AboutDialog(QWidget *parent)
     // Title
     m_ui->labelName->setText(QString("<b><h2>qBittorrent " QBT_VERSION " (%1-bit)</h2></b>").arg(QT_POINTER_SIZE * 8));
 
-    m_ui->logo->setPixmap(Utils::Gui::scaledPixmapSvg(":/icons/skin/qbittorrent-tray.svg", this, 32));
+    m_ui->logo->setPixmap(Utils::Gui::scaledPixmapSvg("AboutDialog.Logo", this, 32));
```

Comment:
I strongly dislike it in this way.
You need to merge `Utils::Gui::scaledPixmapSvg()` and `Utils::Gui::scaledPixmap()`. Also the first parameter shouldn't be changed to iconId. You just should to call it as `Utils::Gui::scaledPixmapSvg(UIThemeManager::getIconPath("AboutDialog.Logo"), this, 32)`

Reply:
Or (even better, IMO) keep `getIconPath()` private and implement `UIThemeManager::getPixmap()` that encapsulates all needed logic.

Reply:
Did you mean by following?
```
QPixmap scaledPixmap(const QPixmap& pixmap, const QWidget *widget, int height = 0);
QPixmap scaledPixmapSvg(const QPixmap &pixmap, const QWidget *widget, int baseHeight);
```

If yes, then `scaledPixmapSvg` requires to have some key to cache, so will it be better to move b
`scaledPixmap` and `scaledPixmapSvg` both inside the `UIThemeManager`.

Reply:
Did you mean by following?
```
QPixmap scaledPixmap(const QPixmap& pixmap, const QWidget *widget, int height = 0);
QPixmap scaledPixmapSvg(const QPixmap &pixmap, const QWidget *widget, int baseHeight);
```

If yes, then `scaledPixmapSvg` requires to have some key to cache, so will it be better to move b
`scaledPixmap` and `scaledPixmapSvg` both inside the `UIThemeManager`.

Reply:
I mean you should implement only one of the following:
```c++
QPixmap UIThemeManager::getPixmap(const QString &id, const QWidget *widget, int baseHeight);
QPixmap Utils::Gui::scaledPixmap(const QPixmap &pixmap, const QWidget *widget, int baseHeight);
```

The client code should know nothing about the type of some particular icon/pixmap (it can be different in different theme). So the function should determine it itself.

Reply:
I tried implementing `UIThemeManager::getPixmap` because then everything will be in one place.I implement this on the basis of scaledPixmapSvg. Can you check now?